data/items.js

```diff
@@ -3117,6 +3117,22 @@ let BattleItems = {
 		desc: "Evolves Gloom into Vileplume, Weepinbell into Victreebel, Exeggcute into Exeggutor or Alolan Exeggutor, Eevee into Leafeon, Nuzleaf into Shiftry, and Pansage into Simisage when used.",
 		shortDesc: "Evolves certain species of Pokemon when used.",
 	},
+	"leek": {
+		id: "leek",
+		name: "Leek",
+		fling: {
+			basePower: 60,
+		},
+		spritenum: 475, //TODO: Sprite, using Stick as a placeholder
+		onModifyCritRatio(critRatio, user) {
+			if (user.baseTemplate.species === 'Farfetch\'d' || user.baseTemplate.species === 'Farfetch\'d-Galar' || user.baseTemplate.species === 'Sirfetch\'d') {
```

Comment:
```suggestion
			if (["Farfetch'd", "Sirfetch'd"].includes(user.baseTemplate.baseSpecies)) {
```

Reply:
Also are we sure this is `baseTemplate` and not `template`?

Reply:
All other implementations of species-specific items (Light Ball, Soul Dew) use ``baseTemplate``